env.example

```diff
@@ -3,3 +3,7 @@ NODE_ENV=development
 
 # PORT is the port used by the web server
 PORT=8080
+
+# INACTIVETIME is the period (millisecond) of inactivity
```

Comment:
Can we do this in some unit other than `ms` and you calculate your `ms` based on that?  I'd suggest weeks, months, or days are more understandable here.

Reply:
Let's do days. I think that's a denomination that's easy enough to understand.